File: java/src/main/java/de/gdata/vaas/messages/Verdict.java
package de.gdata.vaas.messages;

import com.google.gson.annotations.SerializedName;

public enum Verdict {
    @SerializedName("Unknown")
    UNKNOWN,
    @SerializedName("Clean")
    CLEAN,
    @SerializedName("Malicious")
    MALICIOUS,
    @SerializedName("Pup")
    PUP
}

File: golang/vaas/pkg/messages/verdict.go
// Package messages provides structures for handling communication messages between the client and the VaaS server.
package messages

// Verdict represents different verdict outcomes.
type Verdict string

// Verdict outcomes.
const (
	Clean     Verdict = "Clean"
	Unknown   Verdict = "Unknown"
	Malicious Verdict = "Malicious"
	Pup       Verdict = "Pup"
	Error     Verdict = "Error"
)
